Leading `;` causes a panic

When the line is started with `;`, the following panic is occurred.

```ruby
» ;
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x40f99fa]

goroutine 1 [running]:
github.com/goby-lang/goby/compiler/bytecode.(*Generator).compileExpression(0xc4201aefc0, 0xc4201da000, 0x0, 0x0, 0xc4201af050, 0xc4201b0640)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/compiler/bytecode/expression_generation.go:9 +0x3a
github.com/goby-lang/goby/compiler/bytecode.(*Generator).compileStatement(0xc4201aefc0, 0xc4201da000, 0x4820240, 0xc42000c080, 0xc4201af050, 0xc4201b0640)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/compiler/bytecode/statement_generation.go:40 +0x334
github.com/goby-lang/goby/compiler/bytecode.(*Generator).compileStatements(0xc4201aefc0, 0xc420052050, 0x1, 0x1, 0xc4201af050, 0xc4201b0640)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/compiler/bytecode/statement_generation.go:22 +0xd0
github.com/goby-lang/goby/compiler/bytecode.(*Generator).GenerateInstructions(0xc4201aefc0, 0xc420052050, 0x1, 0x1, 0x1, 0xc42001c110, 0xd)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/compiler/bytecode/generator.go:54 +0x5b
github.com/goby-lang/goby/igb.StartIgb(0x457257f, 0x5)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/igb/repl.go:229 +0xcf4
main.main()
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/goby.go:29 +0x954
```

I found a similar issue. The continuous semicolon `;` also causes a panic when 3 or more: 

```ruby
» a=1; -a
#» 0    # invalid: should be -1
» a=1;; -a
#» -1   # works and valid
» a=1;;; -a
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x40f99fa]

goroutine 1 [running]:
github.com/goby-lang/goby/compiler/bytecode.(*Generator).compileExpression(0xc4201a1500, 0xc420214000, 0x0, 0x0, 0xc4201a1590, 0xc4201a46c0)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/compiler/bytecode/expression_generation.go:9 +0x3a
github.com/goby-lang/goby/compiler/bytecode.(*Generator).compileInfixExpression(0xc4201a1500, 0xc420214000, 0xc4201aa300, 0xc4201a1590, 0xc4201a46c0)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/compiler/bytecode/expression_generation.go:265 +0xce
github.com/goby-lang/goby/compiler/bytecode.(*Generator).compileExpression(0xc4201a1500, 0xc420214000, 0x4820420, 0xc4201aa300, 0xc4201a1590, 0xc4201a46c0)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/compiler/bytecode/expression_generation.go:47 +0xc3a
github.com/goby-lang/goby/compiler/bytecode.(*Generator).compileStatement(0xc4201a1500, 0xc420214000, 0x4820240, 0xc4201ae0c0, 0xc4201a1590, 0xc4201a46c0)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/compiler/bytecode/statement_generation.go:40 +0x334
github.com/goby-lang/goby/compiler/bytecode.(*Generator).compileStatements(0xc4201a1500, 0xc4201ae100, 0x2, 0x2, 0xc4201a1590, 0xc4201a46c0)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/compiler/bytecode/statement_generation.go:22 +0xd0
github.com/goby-lang/goby/compiler/bytecode.(*Generator).GenerateInstructions(0xc4201a1500, 0xc4201ae100, 0x2, 0x2, 0x9, 0xc4201c41c0, 0x15)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/compiler/bytecode/generator.go:54 +0x5b
github.com/goby-lang/goby/igb.StartIgb(0x457257f, 0x5)
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/igb/repl.go:229 +0xcf4
main.main()
	/Users/hachi8833/deve/golang/gopath/sys/src/github.com/goby-lang/goby/goby.go:29 +0x954
```

--- a/compiler/bytecode/expression_generation.go
+++ b/compiler/bytecode/expression_generation.go
@@ -6,6 +6,10 @@ import (
 )
 
 func (g *Generator) compileExpression(is *InstructionSet, exp ast.Expression, scope *scope, table *localTable) {
+	if exp == nil {
+		return
+	}
+
 	sourceLine := exp.Line()
 	switch exp := exp.(type) {
 	case *ast.Constant:

--- a/compiler/bytecode/statement_generation.go
+++ b/compiler/bytecode/statement_generation.go
@@ -22,6 +22,11 @@ func (g *Generator) compileStatements(stmts []ast.Statement, scope *scope, table
 		g.compileStatement(is, statement, scope, table)
 	}
 
+	// empty input so no statement is given
+	if len(stmts) == 0 {
+		return
+	}
+
 	g.endInstructions(is, stmts[len(stmts)-1].Line())
 	g.instructionSets = append(g.instructionSets, is)
 }

